control/stabilisation.h

```diff
@@ -77,11 +77,12 @@ typedef enum
 typedef struct 
 {
 	float rpy[3];								///< roll pitch yaw rates/angles
+	float gimbal_rpy[3];						///< roll pitch yaw rates/angles for the gimbal

```

Comment:
The gimbal will not be used by everyone, so I do not think it is a good idea to add it to this structure. You can use other structs that are already defined like `attitude_command_t`